gulp/modules/ErrorHandling.js

```diff
@@ -186,54 +184,7 @@ var ErrorHandling = function() {
         });
     };
     /**
-     * LESSHint error handler
```

Comment:
Same as above: even if we aren't adding SASS linting here, it's super important that we do add it shortly.